Source/WTF/wtf/text/StringCommon.h

```diff
@@ -609,101 +607,50 @@ ALWAYS_INLINE const uint16_t* find16(const uint16_t* pointer, uint16_t character
     // If the pointer is unaligned to 16bit access, then SIMD implementation does not work. But ARM64 allows unaligned access.
     // Fallback to a simple implementation. We also use it for smaller memory where length is less than 16.
     constexpr size_t thresholdLength = 32;
```

Comment:
Maybe we should decrease the threshold? I suppose it's not part of your changes but I'd be surprised if 16-31 byte text isn't profitable to vectorize, since our alternative is a byte-by-byte loop.

Reply:
Yeah, for now, this patch is keeping this threshold. But we could make this smaller after this.